pkg/descheduler/policyconfig_test.go

```diff
@@ -878,9 +878,6 @@ profiles:
       filter:
         enabled:
           - "DefaultEvictor"
-      evict:
-        enabled:
-          - "DefaultEvictor"
```

Comment:
no `preEvictionFilter` here? Or is that already tested?

Reply:
Dropped the `filter` ep as well. `TestDecodeDefaults` provides testing for the missing eps.